lib/ansible/modules/windows/win_shortcut.ps1

```diff
@@ -26,7 +26,7 @@ $ErrorActionPreference = "Stop"
 $params = Parse-Args $args -supports_check_mode $true
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
 
-$src = Get-AnsibleParam -obj $params -name "src" -type "path"
+$src = Get-AnsibleParam -obj $params -name "src"
```

Comment:
Yes, so that is clearly a problem. Unless we decide -type "path" should also accept URLs.
I prefer not to, and to be honest, I prefer to have 2 mutual-exclusive parameters over a single one.

But it's too late now to do the right thing :-(